velox/common/encode/Base64.h

```diff
@@ -122,8 +95,18 @@ class Base64 {
       size_t src_len,
       char* dst,
       size_t dst_len,
-      const ReverseIndex& table,
-      bool include_pad);
+      const ReverseIndex& table);
+
+ public:
+  // Padding character used in encoding
+  constexpr static char kBase = 64;
```

Comment:
I am not able to follow why this char variable is used, could you please clarify?

Reply:
The idea was to use less space in the memory, but it won't serve the purpose here, corrected it.